### ISSUE TITLE
requiring main freezes on ruby 2.1

### ISSUE THREAD

**Issue by martinos** [ahoward/main#35]
When I just require main in a ruby program and I am at the root of my home dir, it freezes my Ruby process. 

In order to reproduce I can:

```
cd ~
ruby -r main -e ''
```

This does not happen when I am in subdirectory. I tried this on another machine and I don't have the issue.

I am on OS X 10.9.2 and using the main-5.3.2 version.

Here is a backtrace that I get when I send a SIGINT:

https://gist.github.com/martinos/9357140

**Comment by prasadsurase** [comment]
@martinos Is the issue solved? I wasnt able to reproduce the issue on OSX(10.11.6 (15G31)), ruby 2.2.2p95 and main 5.3.2.

**Comment by martinos** [comment]
No, I haven't solved that one. I stopped using the gem since it was a show stopper. I tried it on my new computer and this issue is not existing.

**Comment by ahoward** [comment]
cannot repro
